gnd/src/main/java/com/google/android/gnd/ui/offlinearea/OfflineAreasFragment.java

```diff
@@ -21,14 +21,15 @@
 import android.view.View;
 import android.view.ViewGroup;
 import androidx.annotation.Nullable;
+import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
 import butterknife.BindView;
 import com.google.android.gnd.MainActivity;
 import com.google.android.gnd.R;
 import com.google.android.gnd.databinding.OfflineAreasFragBinding;
 import com.google.android.gnd.inject.ActivityScoped;
 import com.google.android.gnd.ui.common.AbstractFragment;
-import com.google.android.gnd.ui.common.TwoLineToolbar;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Can we use `androidx.*` for consistency?

Reply:
Good catch thanks! 34b3dfa